website/views/security.py

```
@@ -0,0 +1,243 @@
+        cache.set(key, 1, CSV_RATE_LIMIT_WINDOW)
+        return False
+
+    return count > CSV_RATE_LIMIT_MAX_CALLS
+
+
+def _escape_csv_formula(value):
+    """Escape leading formula characters to mitigate CSV formula injection."""
+    if not isinstance(value, str):
+        return value
+
+    original = value
```

Ruff (0.14.7)

Local variable `original` is assigned to but never used

Remove assignment to unused variable `original`

(F841)